build.sh

```diff
@@ -241,13 +241,18 @@ case $OSName in
 
     Linux)
         __HostOS=Linux
+        __OSVersion=$(lsb_release -rs)
```

Comment:
Can you remove this and use `$VERSION_ID` instead (it will be defined when we source `/etc/os-release` later)